portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetViews.testcase

```diff
@@ -84,12 +84,31 @@ definition {
 	}
 
 	@description = "Confirm that the user can cancel the creation of a data set view"
-	@ignore = "Test Stub"
 	@priority = 5
 	test NotCreatedAfterCancel {
+		task ("When The user goes to add Datasets") {
+			LexiconEntry.gotoAdd();
+		}
+
+		task ("And fills the field name with name 'Test Dataset'") {
+			PortletEntry.inputName(name = "Test Dataset");
+		}
 
-		// TODO NotCreatedAfterCancel pending implementation JQuery enabled
+		task ("And choose the Provider called channel ") {
+			FrontendDataSetAdmin.selectProvider(key_type = "Channel");
+		}
+
+		task ("And clicking on the Cancel button ") {
+			Click(
+				key_text = "Cancel",
+				locator1 = "Button#ANY_SECONDARY");
+		}
 
+		task ("Then Confirm that the dataset was not created") {
+			VerifyElementNotPresent(
```

Comment:
Hey could you use?
```suggestion
			AssertElementPresent(
```

Reply:
@mateusralv I tried to run with verifyElementPresent but it failed

Reply:
Nops because `VerifyElementNotPresent` can return false positives and using the macro would repeat what was done before. You can cp these two (1ccf831a0e46f89e779baf69086b6c6645555ab3 d2b24b84fd4918cb361e963bc2203ff200c7c7f2)commits and do it this way: 
```suggestion
				AssertElementPresent(
					key_itemName = "Test Dataset",
					locator1 = "FrontendDataSetAdmin#TABLE_CELL");
```

Reply:
Yes, I put the AssertElementNotPresent and it worked